notebooks/community/ml_ops/stage6/get_started_with_tf_serving_tabular.ipynb

```diff
@@ -0,0 +1,1105 @@
+{
```

Comment:
<p>TThe &gt; The</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>